components/python/salt/salt-PYVER.p5m

```diff
@@ -0,0 +1,1653 @@
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/netapi/rest_tornado/saltnado_websockets.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/netapi/rest_wsgi.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/output/__init__.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/output/highstate.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/output/json_out.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/output/key.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/output/nested.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/output/newline_values_only.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/output/no_out.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/output/no_return.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/output/overstatestage.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/output/pony.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/output/pprint_out.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/output/profile.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/output/progress.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/output/raw.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/output/table_out.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/output/txt.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/output/virt_query.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/output/yaml_out.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/payload.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/pillar/__init__.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/pillar/cmd_json.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/pillar/cmd_yaml.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/pillar/cmd_yamlex.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/pillar/cobbler.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/pillar/confidant.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/pillar/consul_pillar.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/pillar/csvpillar.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/pillar/digicert.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/pillar/django_orm.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/pillar/ec2_pillar.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/pillar/etcd_pillar.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/pillar/extra_minion_data_in_pillar.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/pillar/file_tree.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/pillar/foreman.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/pillar/git_pillar.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/pillar/gpg.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/pillar/hg_pillar.py
+file path=usr/lib/python$(PYVER)/vendor-packages/salt/pillar/
```

Comment:
Please, create mediated symlinks fot these man pages (look at barman for example)

Reply:
Done